artifactory/1.11/artifactory-lb.md

```diff
@@ -0,0 +1,50 @@
+# Setting up Artifactory-lb on DC/OS
+
+## Prerequisites
+
+- DC/OS 1.11 or later with at least one public agent
+- [DC/OS CLI installed](https://docs.mesosphere.com/1.11/cli/install/) and
+  configured to use your cluster
+- A running installation of [Artifactory Pro](artifactory-pro.md) or
+  [Artifactory Enterprise](artifactory-enterprise.md) on DC/OS
+
+
+## Setting up Artifactory-lb:
+
+1. Create a new file called `artifactory-lb-options.json`, with the following
+   content:
+
+```
+{
+  "service": {
+    "name": "artifactory-lb"
+  },
+  "artifactory": {
+    "name": "artifactory"
+  }
+}
+```
+
+If you customised the name of Artifactory when installing it, please change
+`artifactory.name`.
+
+2. Run this command to install Artifactory-lb:
+
+```
+dcos package install --options=artifactory-lb-options.json artifactory-lb
+```
+
+## Accessing Artifactory
+
+To access Artifactory, navigate to the DC/OS public agent where Artifactory-lb
+is running:
+
+![Artifactory UI](img/Artifactory_UI_Home.png)
+
```

Comment:
Your screenshot shows "Welcome, admin" but I haven't logged in. Is there a default username/password for it?

Reply:
Will change the screenshot to Artifactory_UI.png ... which will shows the page for first time access. First time access page will take the user through the admin user and Artifactory setup.